lib/contestant.rb

```diff
@@ -0,0 +1,26 @@
+class Contestant
+  attr_reader :first_name, :last_name,
```

Comment:
Since we only ever call the method `full_name` we don't need attr_readers for `:first_name` and `:last_name`